tools/import-image-tests

```diff
@@ -63,9 +63,16 @@ def load_test(name, fullpath) -> dict:
     return test
 
 
-def export_test(test, target):
+def export_test(test, target, manifest_only):
     name = test["id"]
     fullpath = os.path.join(target, name + ".json")
+    if manifest_only and os.path.exists(fullpath):
+        # load existing
+        with open(fullpath, "r", encoding="utf-8") as f:
+            base = json.load(f)
+        # only replace the manifest in the base
+        base["manifest"] = test["manifest"]
```

Comment:
What else is could be in the file? The image info? Wouldn't the manifest and image-info be out of sync then?

Reply:
So in each database file you have:
* the manifest
* the image-info
The update process requires to first update the manifest, then to send that to CI, at the end of the CI job, the commit is amended to integrate the image-info part of it. So it's "out of sync" only on CI, and when it lands as a PR they must be synced. And I think that's part of the review process to ensure that a change in a manifest should result in a change in the image-info side of the DB.

Reply:
though we could imagine a process in a follow up that tries to ensure that every changed manifest did result in a changed image-info. But it'll need some thinking.

Reply:
So `image-info` on an install fails? or is just not very useable? I should know, but I forgot ;)